theano/gpuarray/tests/test_abstractconv.py

```diff
@@ -252,3 +252,12 @@ def setUp(self):
         self.constant_tensor = gpuarray.array(
             numpy.zeros((3, 5, 7, 11), dtype='float32'),
             context=get_context(test_ctx_name))
+
+
+class TestConv2dTranspose(test_abstract_conv.TestConv2dTranspose):
+    mode = mode_with_gpu
+
+    def test_interface(self):
+        if not dnn_available(test_ctx_name):
+            raise SkipTest(dnn_available.msg)
```

Comment:
I think we have CorrMM in the new back-end now, so the test should still be able to run without cuDNN.
In fact, CorrMM is currently the only way of having filter_dilation and subsampling at the same time.